### PR TITLE
Add Queue job to backend

### PR DESCRIPTION
**Pull request by Desvelao**
Hi team, this PR resolves:
- Merge backend branch for ReactJS into `4.0-7.10`
- Includes the Queue job that is missing in `4.0-7.10`

### CODE REVIEW

**Review by frankeros — APPROVED** [review]
LGTM!

File: server/controllers/wazuh-api.ts [hunk 1]
```diff
@@ -967,14 +964,13 @@ export class WazuhApiCtrl {
...
-          method,
-          path,
-          data,
-          options,
+        addJobToQueue({
+          startAt: new Date(Date.now() + delay),
+          run: async () => {
+            try{
+              await context.wazuh.api.client.asCurrentUser.request(method, path, data, options);
+            }catch(error){};
```
Comment (gabiwassan) [inline server/controllers/wazuh-api.ts:972]: Here we should log a possible error or propagate ?

**Review by gabiwassan — CHANGES_REQUESTED** [review]
Just a simple comment, Great job!!! 💪🏼

File: server/controllers/wazuh-api.ts (hunk 1 above)
Comment (Desvelao) [reply server/controllers/wazuh-api.ts:972]: Good idea! 👍

**Review by gabiwassan — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**